schedule/yam/agama.yaml

```diff
@@ -5,3 +5,4 @@ description: >
 schedule:
   - installation/bootloader_start
   - yam/agama/agama_is_visible
+  - yam/agama/patch_agama_tests
```

Comment:
you need to put it before agama_is_visible as mentioned in previous comment.

Reply:
maybe it will not work like that...now I have doubts, let's merge then, anyway we are going to refactor this.